Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -860,6 +860,110 @@ DECLARE_SOA_TABLE(Decays3Body, "AOD", "DECAY3BODY", //! Run 2 cascade table
 using Decays3Body = Decays3Body; //! this defines the current default version
 using Decay3Body = Decays3Body::iterator;
 
+namespace strangenesstracking
+{
+DECLARE_SOA_INDEX_COLUMN_FULL(ITSIdx, Idx, int, Tracks, "");                    //! ITS index
+DECLARE_SOA_INDEX_COLUMN(Cascade, cascadeIdx);                                  //! Cascade index
+DECLARE_SOA_INDEX_COLUMN(V0, v0Idx);                                            //! V0 index
+DECLARE_SOA_INDEX_COLUMN_FULL(Decay3Body, decay3BodyIdx, int, Decays3Body, ""); //! Decay 3 body index
+
+DECLARE_SOA_COLUMN(DecayX, decayX, float);             //! X coordinate of decay vertex
+DECLARE_SOA_COLUMN(DecayY, decayY, float);             //! Y coordinate of decay vertex
+DECLARE_SOA_COLUMN(DecayZ, decayZ, float);             //! Z coordinate of decay vertex
+DECLARE_SOA_COLUMN(XiMass, xiMass, float);             //! Xi mass
+DECLARE_SOA_COLUMN(OmegaMass, omegaMass, float);       //! Omega mass
+DECLARE_SOA_COLUMN(H3Lmass, h3Lmass, float);           //! H3L mass
+DECLARE_SOA_COLUMN(H4Lmass, h4Lmass, float);           //! H4L mass
+DECLARE_SOA_COLUMN(He4Lmass, he4Lmass, float);         //! He4L mass
```

Comment:
During the WP14 meeting you did not show all these masses but only two of them if I am not mistaken. Is there no way to some of these from each other?

Reply:
To my understanding at the moment it is anyway incomplete because we need the compressed (stored) and uncompressed (through the spawner table). TracksCov vs. StoredTracksCov. These are a lot of additions to the data model, so I vote for having it in the Tracks table.

The track selection already checks the track type and yes, other service tasks, should do too (and then fill empty values for the entries skipped).

Reply:
Ah, I realize I forgot to comment on the individual masses being present: I believe this is a reasonable thing to do. A recalculation would require the daughter tracks, which would blow up the size much more than just storing a few masses... Anyway the exact recalculation of the masses also needs to have its full effect studied and documented later but the data model change with the additional masses looks very reasonable to me!

Reply:
Ok, thanks a lot for your comments. I drafted the P.R. again and I am now looking for the best way to modify the AODProducer to add the strange tracks into the full track table